### ISSUE TITLE
Test the app on windows

### ISSUE THREAD

**Issue by wvffle** [wvffle/minecraft-packet-debugger#3]
Does everything work on windows?

**Comment by Rattlyy** [comment]
nope

**Comment by Rattlyy** [comment]
C:\Users\me>mcpd                                                                                                     C:\Users\me\AppData\Roaming\npm\node_modules\mcpd\lib\config.js:56                                                     save(defaultConfig).then(({ error }) => {                                                                               ^                                                                                                                                                                                                                                             ReferenceError: save is not defined                                                                                         at Object.<anonymous> (C:\Users\me\AppData\Roaming\npm\node_modules\mcpd\lib\config.js:56:3)                         at Module._compile (internal/modules/cjs/loader.js:1158:30)                                                             at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)                                               at Module.load (internal/modules/cjs/loader.js:1002:32)                                                                 at Function.Module._load (internal/modules/cjs/loader.js:901:14)                                                        at Module.require (internal/modules/cjs/loader.js:1044:19)                                                              at require (internal/modules/cjs/helpers.js:77:18)                                                                      at Object.<anonymous> (C:\Users\me\AppData\Roaming\npm\node_modules\mcpd\lib\utils.js:3:20)                          at Module._compile (internal/modules/cjs/loader.js:1158:30)                                                             at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)

**Comment by wvffle** [comment]
hmm
What's your node version?

**Comment by Rattlyy** [comment]
C:\Users\me>node -v                                                                                                  
v12.16.1

**Comment by wvffle** [comment]
hmm
seems to work well on 13.11.0 but I'll fix it

**Comment by Systemwartung** [comment]
OK thankw

**Comment by wvffle** [comment]
@zRattly update mcpd and check now ;)

**Comment by Rattlyy** [comment]
trying

**Comment by Rattlyy** [comment]
Tryed connecting, crashed my minecraft lol

**Comment by wvffle** [comment]
For now 1.12.2 is hardcoded ;/

**Comment by Rattlyy** [comment]
oh, tryed joining with 1.8.9 lol

**Comment by Rattlyy** [comment]
but there are some errors in gui and console

**Comment by Rattlyy** [comment]
{"statusCode":500,"code":"ENOENT","error":"Internal Server Error","message":"ENOENT: no such file or directory, open 'C:\\Users\\giova\\public\\index.html'"} this is the web error

**Comment by Rattlyy** [comment]
UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir 'C:\Users\me\AppData\Roaming\mcpd\Config'                                                                                                          at Object.mkdirSync (fs.js:840:3)                                                                                       at C:\Users\giova\AppData\Roaming\npm\node_modules\mcpd\lib\config.js:59:12                                         (node:4568) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)                                                     (node:4568) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code. (this is in the console)

**Comment by wvffle** [comment]
@zRattly should've fixed those two errors

**Comment by Ethan-Vann** [comment]
still an issue in the latest release
![Error](https://user-images.githubusercontent.com/22735513/78004007-8580d680-7307-11ea-84be-4f6fd66324a1.PNG)

**Comment by wvffle** [comment]
@Ethan-Vann, there's an open PR for that #10

**Comment by wvffle** [comment]
@Ethan-Vann, @zRattly any news after #10 has been merged?

**Comment by Ethan-Vann** [comment]
should have tried this before. just switched to arch to test back then. is the npm build outdated? grabbing with npm produces same error but getting from source is working now

**Comment by wvffle** [comment]
Yup, use the github version

**Comment by Saiv46** [comment]
Now we need to update NPM package and close this issue

**Comment by wvffle** [comment]
@Saiv46 so you confirm it working?

**Comment by Saiv46** [comment]
Yep.

**Comment by Saiv46** [comment]
@wvffle Fixed package version still not published :/

**Comment by wvffle** [comment]
Thanks for reminding me that. Currently I have no spare time for mineflayer related projects. Lots of work and also studies :/

I'll publish it in a minute

**Edit:** already done, @Saiv46
